src/services/gateways/aws.ts

```diff
@@ -1045,19 +875,15 @@ export class AWS {
   }
 
   async getLoadBalancer(arn: string) {
-    const result = await this.elbClient.send(
-      new DescribeLoadBalancersCommand({ LoadBalancerArns: [arn], })
-    );
+    const result = await this.elbClient.describeLoadBalancers({ LoadBalancerArns: [arn], });
     return result?.LoadBalancers?.[0];
   }
 
   async deleteLoadBalancer(arn: string) {
-    await this.elbClient.send(
-      new DeleteLoadBalancerCommand({ LoadBalancerArn: arn, })
-    );
+    await this.elbClient.deleteLoadBalancer({ LoadBalancerArn: arn, });
     // We wait it is completely deleted to avoid issues deleting dependent resources.
-    const input = new DescribeLoadBalancersCommand({ LoadBalancerArns: [arn], });
-    await createWaiter<ElasticLoadBalancingV2Client, DescribeLoadBalancersCommand>(
+    const input: DescribeLoadBalancersCommandInput = { LoadBalancerArns: [arn], };
+    await createWaiter<ElasticLoadBalancingV2, DescribeLoadBalancersCommandInput>(
```

Comment:
Are you sure this is ok? For the waiters I think we still need to pass the old syntax

Reply:
Yeah, it still fulfills the interface and it works!